.github/workflows/tests.yml

```
@@ -1,112 +1,238 @@
-    name: Build Check
+    name: Build (${{ matrix.os }}/${{ matrix.arch }})
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        os: [linux, windows, darwin]
+        os: [linux, darwin]
         arch: [amd64, arm64]
+        exclude:
+          # Windows builds require special handling for CGO
+          - os: windows
```

actionlint (1.7.8)

value "windows" in "exclude" does not match in matrix "os" combinations. possible values are "linux", "darwin"

(matrix)